packages/core/components/FileDetails/FileAnnotationList.tsx

```diff
@@ -65,12 +72,22 @@ export default function FileAnnotationList(props: FileAnnotationListProps) {
                 return accum;
             }
 
-            const annotationValue = annotation.extractFromFile(fileDetails);
+            let annotationValue = annotation.extractFromFile(fileDetails);
             if (annotationValue === Annotation.MISSING_VALUE) {
                 // Nothing to show for this annotation -- skip
                 return accum;
             }
 
+            if (annotation.name === AnnotationName.LOCAL_FILE_PATH) {
+                if (localPath === null) {
+                    // localPath hasn't loaded yet
```

Comment:
Local path must exist because there is an annotation with the name Local File Path (line 81): if a file is not present on the Vast, there is no Local File Path annotation.

Reply:
Added this elaboration in [29423b0](https://github.com/AllenInstitute/biofile-finder/pull/370/commits/29423b0518569239c52c6c0b912f6fb661616d01)